/*#global testerror "Err0401"
//func toto() => "toto"

//#assert(false, toto())

#test
{
    defer #err (a += 1
}*/
